### PR TITLE
add about section and work section

### PR DESCRIPTION
**Pull request by HanaSabih**
add and style work section and about section

### CODE REVIEW

**Review by ha-manel — COMMENTED** [review]
# STATUS: INVALID 🔴 
Hi @HanaSabih 
I'm sure you put a lot of effort into this project but this PR is invalid because your README file is missing a lot of important sections😓Kindly copy them from this [template](https://github.com/microverseinc/readme-template) and fill out the sections with information about your project 👍  

Once you're ready please submit the PR for review again 👍 
Best of luck!

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -33,5 +33,155 @@ <h1>
...
+              <li><span>Ruby on rails</span></li>
+              <li><span>RSpec </span></li>
+              <li><span>Capybara</span></li>
+              <li><span>Selenium</span></li>
+            </ul>
+          </div>
+        </div>
+        <div class="cardabt">
+          <div class="imageabout">
+            <img src="images/icons/about3.png" alt="" />
```
Comment (mgmediaweb) [inline index.html:171]: [OPTIONAL] Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

File: index.html [hunk 2]
```diff
@@ -33,5 +33,155 @@ <h1>
...
+              <li><span>JavaScript</span></li>
+              <li><span>Ruby</span></li>
+              <li><span>Html </span></li>
+              <li><span>CSS</span></li>
+            </ul>
+          </div>
+        </div>
+        <div class="cardabt">
+          <div class="imageabout">
+            <img src="images/icons/about3.png" alt="" />
```
Comment (mgmediaweb) [inline index.html:156]: [OPTIONAL] Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

File: index.html [hunk 3]
```diff
@@ -33,5 +33,155 @@ <h1>
...
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+        <button>Get My Resume</button>
+      </div>
+      <div class="card_about">
+        <div class="cardabt">
+          <div class="imageabout">
+            <img src="images/icons/about3.png" alt="" />
```
Comment (mgmediaweb) [inline index.html:142]: [OPTIONAL] Kindly, I suggest that for accessibility every time you use an "IMG" you indicate an appropriate "ALT" to describe the content of that image

**Review by mgmediaweb — APPROVED** [review]
Hi @HanaSabih , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approved5

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 
- Clean and tidy code. ✔️ 
- Good job following the design. ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**